CONTRIBUTING.md

```diff
@@ -1,14 +1,14 @@
 # Contributing to `code-quality-tools`
 
-First of all, thanks for your contribution to this project! ❤️ Here are some tips how to make your contributing efforts efficient and eventually accepted & merged.
+Before all anything, thanks for your contribution to this project! ❤️ Here are some tips how to make your contributing efforts efficient and eventually accepted & merged.
 
 ## General usage
 
 This project uses `Makefile` for managing various chores, like dependency installation, testing, linting etc. Make sure you run `make` right after you clone the repository - it will set you up with everything needed to get started. Once you are done with that, have a look at what commands (targets) are available for you to run using `make` - `make install`, `make lint` etc.
 
 ## Commit messages
 
-All commits you make should adhere to our commit guidelines. We use [conventional commits][conventional-commits] strategy with our [slight modification of used keywords][lmc-conventional-commits]. This is later used during release phase to determine how to bump the packages' version numbers based on commit history. 🚀
+All commits you make should adhere to our commit guidelines. We use [conventional commits][conventional-commits] strategy with our [slight modification of used keywords][lmc-conventional-commits]. This is later used during release phase to determine how to bump the package’s version numbers based on commit history. 🚀
```

Comment:
I think the plural was actually correct. Because a package can only have one version number, but [many] packages have their version numbers. 🙂

Reply:
Thanks, fixed.